README.md

```diff
@@ -66,7 +66,7 @@ isFetchableUrlInNode('https://example.aexp.com/path');
 
 ## preprocessEnvVar( [config] )
 
-Loads the local module map and remote module map for local development.
+Processes environment variables and applies validations, defaults, etc...
```

Comment:
Can we expand on `etc...`? If not I don't think we should include etc as it is a little ambiguous.

Reply:
I updated this in c5a3f1d